packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/overview/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Overview
+---
+
+Okta Event and Inline Hooks use outbound calls, which are received and parsed by an external service to implement additional custom functionality for your Okta implementation.
+
+A secure web server and application is a requirement to implement an Okta Event or Inline Hook, and is referred to as the external service in these guides.
+
+For instructional purposes, the following example guides for Event Hooks and Inline Hook types use the third-party site [Glitch](https://glitch.com) to function as an online external service, one you can use to quickly implement the sample code to preview the functionality of an Event or Inline Hook. You also need a working dev or preview Okta org to configure the hooks and implement the hook functionality.
```

Comment:
This part of the sentence is a bit awkward:

"....function as an online external service, one you can use to quickly implement..."

Can we reword it a bit?

Reply:
Thanks, agree it was a bit clunky. Split into two sentences: 

"For instructional purposes, the following guides for Event Hooks and Inline Hook types use the third-party site [Glitch](https://glitch.com), which functions as an online external service. You can use the example code to quickly implement the hook and preview the functionality. You also need a working dev or preview Okta org to configure the hooks and implement the hook functionality."